service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -17,9 +28,7 @@
 #     1- Get the GCS path from VM metadata (instance/attributes/post-startup-script)
 #     2- Download it to /opt/c2d/post_start.sh
 #     3- Execute /opt/c2d/post_start.sh
-#     4- Set the VM guest attribute "notebooks/handle_post_startup_script" to "DONE"
-#
-#   Note that the guest attribute is set to DONE whether the script runs successfully or not.
```

Comment:
keeo this line

Reply:
I removed this because I thought that this was the old way of how we're setting the startup script status. I now realize that this might have been the behavior of vertexi ai notebooks after the post_start script is executed. Is the latter the case? I couldn't find where these steps are documented.

Reply:
gettign to COMPLETE doesn't mean all the execution succeeded i thought.

Reply:
I thought so, assuming that everything in the script runs sync. The UI also only reads status and message attributes. https://github.com/verily-src/terra-ui/blob/master/src/components/cloud/startupScript.ts#L43-L56

Reply:
ah ok. i didn't realize it has set -o errexit at the top.

Reply:
I stand corrected, UI is indeed querying this attribute as a fallback as well.

Reply:
i'm not asking about the UI fwiw. I didn't know it has `set -o e` at the top to make sure the failed command doesn't fall through.

Reply:
yeah I gotcha, we do have a handler to set the status attribute wherever the script fails. But either way, setting the handle post startup script attribute is a vertex ai behavior that we do read from